packages/react-components/react-tag-picker-preview/src/components/TagPicker/useTagPickerContextValues.ts

```diff
@@ -2,12 +2,64 @@ import * as React from 'react';
 import { TagPickerContextValues, TagPickerState } from './TagPicker.types';
 
 export function useTagPickerContextValues(state: TagPickerState): TagPickerContextValues {
+  const {
+    onOptionClick,
```

Comment:
I think the unused state/types should be cleaned up in a future PR - not blocking just an FYI